plugins/auth-backend/src/providers/resolvers.ts

```diff
@@ -35,3 +35,24 @@ export const commonByEmailLocalPartResolver: SignInResolver<unknown> = async (
     entityRef: { name: localPart },
   });
 };
+
+/**
+ * A common sign-in resolver that looks up the user using their email address
+ * as email of the entity.
+ */
+export const commonByEmailResolver: SignInResolver<unknown> = async (
```

Comment:
i guess we could have been using a more explicit name here too, but it isn't exported, so

Reply:
later man!